src/components/LocationPage/ChartBlock.tsx

```diff
@@ -115,7 +114,7 @@ function ChartBlock(props: {
           ) : (
             <Fragment>
               <ChartDescription>
-                {generateChartDescription(projection, noInterventionProjection)}
+                {metricFutureProjection.renderStatus(props.projections)}
```

Comment:
We should be able to get rid of the special handling for future projections here, but I prefer to do that on a separate PR to keep the diff manageable.

Reply:
👍  to separate PRs! 😁